### PR TITLE
proxy-12sp4 backport for maint 3.2

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/5717

### CODE REVIEW

File: adoc/advanced_topics_proxy_quickstart.adoc [hunk 1]
```diff
@@ -101,58 +99,61 @@ in virtual environments, use the following settings for the virtual machine (VM)
...
-.Registering Proxies
-====
 {productname} Proxy systems are registered as traditional clients or as Salt clients using a bootstrap script.
 A {susemgrproxy} can serve both Traditional and Salt clients.
-====
 
+[IMPORTANT]
+====
+First completely download the channels ({sle} 12 SP4) and then create the activation key.
+Only then you can select the correct child channels.
```
Comment (Loquacity) [inline adoc/advanced_topics_proxy_quickstart.adoc]: Change the wording here so the most important thing comes first:
"Before you can select the correct child channels, ensure you have completely downloaded the channels for {sle} 12 SP4, and created the activation key."

File: adoc/advanced_topics_proxy_quickstart.adoc [hunk 2]
```diff
@@ -162,7 +163,7 @@ For more information, see <<at.manager.proxy.register.saltclients>>.
 [[at.manager.proxy.run.pattern]]
 === Install the [path]``suma_proxy`` pattern
 
-Make sure the [path]``suma_proxy`` pattern version 2.5.1.3 or later is installed using the following command on the proxy as root:
+Either on the sever select the [package]``pattern_suma_proxy`` package for installation, or make sure the [path]``suma_proxy`` pattern is installed using the following command on the proxy as root:
```
Comment (Loquacity) [inline adoc/advanced_topics_proxy_quickstart.adoc]: You can probably remove the 'either' to make this make more sense. Also typo on "server" ;)

File: adoc/advanced_topics_proxy_quickstart.adoc (hunk 1 above)
Comment (keichwa) [reply adoc/advanced_topics_proxy_quickstart.adoc]: ok, but selecting/assigning channels take place while creating the activation key, I think I'd better say:
> Before you can select the correct child channels while creating the activation key, ensure you have completely downloaded the channels for {sle} 12 SP4.

File: adoc/advanced_topics_proxy_quickstart.adoc (hunk 2 above)
Comment (keichwa) [reply adoc/advanced_topics_proxy_quickstart.adoc]: Oops, thanks!

**Comment by keichwa** [comment]
ok, just re-open if there are (newly intro'ed) issues.

**Pull request merged**